uktrade/data-hub-frontend
# Add D&B hierarchy to the company business details page

## Description of change

Add information about D&B hierarchy to the business details page.
 
## Screenshots

![image](https://user-images.githubusercontent.com/4199239/69826650-fbffb180-120b-11ea-96c6-b48f42c98a4a.png)

## Test instructions

Go to `/companies/9aacdbb0-32b3-4dc5-a961-24e6e6c57a46/business-details2` and check if the D&B hierarchy is visible.

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied. 
https://github.com/uktrade/data-hub-frontend/blob/develop/docs/Code%20review%20guidelines.md"
